cx_Freeze/freezer.py

```diff
@@ -740,13 +747,16 @@ def __init__(self, releaseString = None, copyright = None,
         self.values["BUILD_RELEASE_STRING"] = releaseString
         self.values["BUILD_COPYRIGHT"] = copyright
         for constant in constants:
-            parts = constant.split("=")
+            parts = constant.split("=", maxsplit=1)
             if len(parts) == 1:
                 name = constant
                 value = None
             else:
                 name, stringValue = parts
                 value = eval(stringValue)
+            if not _isValidVariableName(name):
+                raise Exception("Invalid constant name in ConstantsModule (\"{}\")".format(name))
+
```

Comment:
_isValidVariableName - for new funcions, you can use pep8 names. Anthony had that intention, and I intend to implement that change.
On the other hand, I think that here you can make a direct comparison and use the ConfigError exception.
if name.isidentifier() or iskeyword(name):
   raise ConfigError("....")

Reply:
Fixed.

I don't normally use PEP8 names, but will try to remember that for this project.